### DISCUSSION TITLE
OT Bad Word

### DISCUSSION THREAD

**Discussion by DVMPT** [kwsch/PKHeX#3822] [General]
Hey there
I’ve been looking through the code but I didn’t find anything about it
The program isn’t letting me save any pk8 or pk9 file with the OT “Pakistan” which has been my OT since 1998.
After exporting the pk file if I reload the file in pkhex or put it back in switch it inputs on the OT Sword. or Violet. depending on the Game Save.
Was there an implementation of a banned name list or something
Before SV came out I could do it but since then I’ve been messing around with the program and I can’t do anything about it

**Comment by Lusamine** [comment]
That sounds like you're hitting an in-game filter of some sort. If you trade any Pokémon that hits the GF name filter, the game will change it to the version name with a period after it.

Are you "exporting" with a trade bot or something? If so, it's subject to trade rules and any other trader will see the filtered name instead.

**Reply by DVMPT** [reply]
Ok but what about catching something and trade in game with another person wouldn’t that suppose to fix the ot too??

**Reply by kwsch** [reply]
Anything with a bad OT/Nickname will be sanitized back to un-nicknamed and a generic OT name.

PKHeX uses the 3DS wordfilter; if "paki" is a disallowed string, then it would explain the behavior you are observing.

**Reply by DVMPT** [reply]
Wow
Ok that new 
Then how could this is only happening now and while menaging save files on 3DS and then to bank and then to home and trade with others never threw that problem before?
Even sysbots never had that problem, why is it hitting this wall now?

**Reply by Lusamine** [reply]
This has nothing to do with sysbot or PKHeX.  If it's new, this is purely GF updating the online trade filter.

Why wasn't it a problem before? Because they didn't have it in the filter before.

**Reply by DVMPT** [reply]
Hummm
I guess now I’ll literally hitting a wall hey
I’ve found the 3DS bad word list. There is a base file in the system access data. It’s there all right

![5AF840AD-7B90-44D3-A5FF-240D90EB9F9F](https://user-images.githubusercontent.com/47702483/222303403-69b473db-4f65-490e-86e9-2680c7a090e9.jpeg)

Well mates thanks for the support and the quick answer and keep up the good work
